src/lib/code-sample/schema.ts

```diff
@@ -79,6 +80,11 @@ export const createCodeSample = (
         request: createPhpRequest(codeSampleDefinition, context),
         response: createPhpResponse(codeSampleDefinition, context),
       },
+      bash: {
+        title: 'Bash',
```

Comment:
```suggestion
        request_syntax: 'bash',
        response_syntax: 'json',
```

Reply:
updated in https://github.com/seamapi/blueprint/pull/58/commits/0b1f37a41acad1ce4a296178b333d1b07f8c20b7